index.html

```diff
@@ -0,0 +1,859 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Design responsive</title>
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" href="./css/styles.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="nav-logo">
+                <img alt="logo-1023px" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+            </div>
+            <div class="nav-bar-content">
+                <div class="topNav">
+                    <ul>
+                        <li><a href="#">News</a></li>
+                        <li><a href="#">Events</a></li>
+                        <li><a href="#">Business</a></li>
+                        <li><a href="#">AMAs</a></li>
+                        <li><a href="#">Spaces</a></li>
+                        <li><a href="#">Terms & Conditions</a></li>
+                    </ul>
+                    <ul class="nav-bar-end">
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Team</a></li>
+                        <li><a href="#">Advertise</a></li>
+                        <li><a href="#">Jobs</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="bottomNav">
+                    <ul>
+                        <li><a href="#">LATEST</a></li>
+                        <li><a href="#">HARD FORK</a></li>
+                        <li><a href="#">PLUGGED</a></li>
+                        <li><a href="#">FUNDAMENTALS</a></li>
+                        <li><a href="#">WORK 2030</a></li>
+                    </ul>
+                    <button>
+						<i class="search-icon"></i>
+					</button>
+                </div>
+            </div>
+        </nav>
```

Comment:
- [ ] your navbar is missing the yellow part

- [ ] you navbar should be on the very top with no spaces on top or on the sides on all sizes

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/91918868-1e9f8900-ecbc-11ea-960f-a215b2abf9e8.png) | ![image](https://user-images.githubusercontent.com/11758151/91918882-29f2b480-ecbc-11ea-8702-d3364db3508c.png) |

- [ ] you need to have a hamburger button on the small screen size

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/91918985-72aa6d80-ecbc-11ea-8428-e2bb60c33b03.png) | ![image](https://user-images.githubusercontent.com/11758151/91919008-81912000-ecbc-11ea-95f4-0350a4cb7a6e.png) |